<ui version="4.0" >
 <class>transactionDialog</class>
 <widget class="QDialog" name="transactionDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>331</width>
    <height>234</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Processing Queue...</string>
  </property>
  <widget class="Line" name="line" >
   <property name="geometry" >
    <rect>
     <x>7</x>
     <y>140</y>
     <width>311</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar" >
   <property name="geometry" >
    <rect>
     <x>17</x>
     <y>160</y>
     <width>291</width>
     <height>21</height>
    </rect>
   </property>
   <property name="value" >
    <number>24</number>
   </property>
  </widget>
  <widget class="QWidget" name="" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>10</y>
     <width>281</width>
     <height>121</height>
    </rect>
   </property>
   <layout class="QGridLayout" >
    <item row="0" column="0" >
     <widget class="QLabel" name="label_2" >
      <property name="text" >
       <string>Icon!!</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" >
     <widget class="QLabel" name="label" >
      <property name="text" >
       <string>Checking transaction validity</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1" >
     <widget class="QLabel" name="label_3" >
      <property name="text" >
       <string>Downloading Packages</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1" >
     <widget class="QLabel" name="label_4" >
      <property name="text" >
       <string>Process queue</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1" >
     <widget class="QLabel" name="label_5" >
      <property name="text" >
       <string>Cleanup</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" >
     <widget class="QLabel" name="label_6" >
      <property name="text" >
       <string>Icon!!</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" >
     <widget class="QLabel" name="label_7" >
      <property name="text" >
       <string>Icon!!</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" >
     <widget class="QLabel" name="label_8" >
      <property name="text" >
       <string>Icon!!</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="" >
   <property name="geometry" >
    <rect>
     <x>12</x>
     <y>190</y>
     <width>311</width>
     <height>32</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" >
    <item>
     <widget class="QPushButton" name="showDetails" >
      <property name="text" >
       <string>More Details...</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer>
      <property name="orientation" >
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" >
       <size>
        <width>71</width>
        <height>30</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="abortTr" >
      <property name="text" >
       <string>Abort</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
